packages/angular-output-target/src/utils.ts

```diff
@@ -85,6 +85,15 @@ export async function readPackageJson(config: Config, rootDir: string) {
   return pkgData;
 }
 
+/**
+ * Creates an import statement for a list of named imports from @angular/core.
+ * @param imports The list of named imports.
+ * @returns The import statement as a string.
+ */
+export const createAngularCoreImportStatement = (imports: string[]) => {
+  return `import { ${imports.join(', ')} } from '@angular/core';`;
```

Comment:
Can we add some protection in the event that `imports` has a length of zero (and a test for it)? Otherwise, I think ATM this generates invalid TS:
```
`import {  } from '@angular/core';`
```